src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/PrivacyNoticesApiServiceImpl.scala

```diff
@@ -104,4 +109,28 @@ final case class PrivacyNoticesApiServiceImpl(
       }
     }
   }
+
+  override def getPrivacyNoticeFromBucket(consentType: String)(implicit
+    contexts: Seq[(String, String)],
+    toEntityMarshallerProblem: ToEntityMarshaller[Problem],
+    toEntityMarshallerFile: ToEntityMarshaller[File]
+  ): Route = {
+    logger.info(s"Retrieving privacy notices for consentType $consentType")
+
+    val result: Future[HttpEntity.Strict] = for {
+      ctype  <- ConsentType.fromValue(consentType).toFuture
+      ppId   <- consentTypeMap.get(ctype).toFuture(PrivacyNoticeNotFoundInConfiguration(consentType))
+      ppUuid <- ppId.toFutureUUID
```

Comment:
please use extended naming to improve readability
```suggestion
      consentType  <- ConsentType.fromValue(consentType).toFuture
      privacyNoticeId   <- consentTypeMap.get(consentType).toFuture(PrivacyNoticeNotFoundInConfiguration(consentType))
      privacyNoticeUuid <- privacyNoticeId.toFutureUUID
```